.github/workflows/alpha-integration-tests.yaml

```diff
@@ -0,0 +1,92 @@
+name: DataRepo Alpha Integration Tests
+env:
+  GOOGLE_APPLICATION_CREDENTIALS: /tmp/jade-dev-account.json
+  GOOGLE_SA_CERT: /tmp/jade-dev-account.pem
+  GOOGLE_CLOUD_PROJECT: terra-datarepo-alpha
+  GOOGLE_CLOUD_DATA_PROJECT: terra-datarepo-alpha-data
+  IT_JADE_API_URL: https://data.alpha.envs-terra.bio
+  PGHOST: 127.0.0.1
+  PGPASSWORD: postgres
+  DB_DATAREPO_URI: "jdbc:postgresql://${PGHOST}:5432/datarepo"
+  DB_STAIRWAY_URI: "jdbc:postgresql://${PGHOST}:5432/stairway"
+on:
+  # Jenkins will kick this job off manually after Alpha is deployed
+  workflow_dispatch: {}
+jobs:
+  alpha-integration-test:
+    runs-on: ubuntu-latest
+    # run a local Postgres container in Docker for the basic check tests
+    services:
+      postgres:
+        image: postgres:11
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    steps:
+    - name: "Checkout code"
+      uses: actions/checkout@v2
+    - name: "Import Vault alpha secrets"
+      uses: hashicorp/vault-action@c8b383ee
+      with:
+        url: ${{ secrets.VAULT_ADDR }}
+        method: approle
+        roleId: ${{ secrets.ALPHA_ROLE_ID }}
+        secretId: ${{ secrets.ALPHA_SECRET_ID }}
```

Comment:
Yes, these are used to cleanup the IAM policies using the Alpha service account.